lms/services/canvas.py

```diff
@@ -0,0 +1,43 @@
+from lms.services.exceptions import CanvasFileNotFoundInCourse
+
+
+class CanvasService:
```

Comment:
Not sure if this is needed, I think it would would give me pause every time I add something that talks to canvas, is is CanvasService? Or CanvasAPIService?

I think the fact that the only public method returns a value from the api service could mean that the boundary is not that clear yet and this could go into the existing one. "CanvasService" is a better name than the current one thought.